test-fixture.html

```diff
@@ -313,9 +315,20 @@
     });
 
   try {
-    document.registerElement('test-fixture', {
-      prototype: TestFixturePrototype
-    });
+    if (window.customElements) {
+      function TestFixture() {
+        if (window.Reflect) {
+          return Reflect.construct(HTMLElement, [], TestFixture)
```

Comment:
`return Reflect.construct(HTMLElement, [], this.prototype.constructor) || this;`

or maybe,

```
var self = ((window.Reflect && Reflect.construct)
   ? Reflect.construct(HTMLElement, [], this.prototype.constructor)
   : HTMLElement.call(this)) || this;
return self;
```